application/src/main/java/org/togetherjava/tjbot/commands/moderation/ReportCommand.java

```diff
@@ -102,26 +105,30 @@ public void onMessageContext(MessageContextInteractionEvent event) {
 
     @Override
     public void onModalSubmitted(ModalInteractionEvent event, List<String> args) {
-        String reportedMessage = args.get(0);
-        String reportedMessageUrl = args.get(1);
-
         long guildID = Objects.requireNonNull(event.getGuild(), "Could not retrieve the guildId.")
             .getIdLong();
-        Optional<TextChannel> modMailAuditLog = getModMailChannel(event.getJDA(), guildID);
+        Optional<TextChannel> modMailAuditLog =
+                handleRequireModMailChannel(event.getJDA(), guildID);
+
+        if (modMailAuditLog.isEmpty()) {
+            return;
+        }
```

Comment:
What happens if the channel is not found? The user does not get a response at all.

Reply:
than its logged as a warning

Reply:
But the user does not see the warning

Reply:
Nop it doesn't

Reply:
Then please add that.

Reply:
Fixed